graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -154,7 +154,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name of the GraphiQL page title.
          */
-        public static final String PAGE_TITLE = PREFIX + ".page-title";
+        public static final String PAGE_TITLE_CONFIG = PREFIX + ".page-title";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.